packages/entities/entities-plugins/src/plugins-endpoints.ts

```diff
@@ -23,12 +23,18 @@ export default {
       edit: '/api/runtime_groups/{controlPlaneId}/plugins/{id}',
       pluginSchema: '/api/runtime_groups/{controlPlaneId}/schemas/plugins/{plugin}',
       validate: '/api/runtime_groups/{controlPlaneId}/v1/schemas/json/plugin/validate',
+      // VFG endpoints
+      entityGetOne: '/api/runtime_groups/{controlPlaneId}/{entity}/{id}',
+      entityGetAll: '/api/runtime_groups/{controlPlaneId}/{entity}?size=1000',
     },
     kongManager: {
       create: '/{workspace}/plugins',
       edit: '/{workspace}/plugins/{id}',
       pluginSchema: '/{workspace}/schemas/plugins/{plugin}',
       validate: '/{workspace}/schemas/plugins/validate',
+      // VFG endpoints
+      entityGetOne: '/{workspace}/{entity}/{id}',
+      entityGetAll: '/{workspace}/{entity}?size=1000',
```

Comment:
```suggestion
      entityGetAll: '/{workspace}/{entity}',
```

When calling `getAll`, `FieldAutoSuggest` has [a pre-defined page size](https://github.com/Kong/public-ui-components/blob/a7e83306e2b303b719219693dc6155f9d132e886/packages/core/forms/src/generator/fields/advanced/FieldAutoSuggest.vue#L222), so here we need to remove `size=1000`, otherwise there'll be two `size` params and Admin API will return 500.